openhands/events/action/message.py

```diff
@@ -7,7 +7,7 @@
 @dataclass
 class MessageAction(Action):
     content: str
-    images_urls: list[str] | None = None
+    image_urls: list[str] | None = None
```

Comment:
If I see this right, to make this name consistent, we could change it in the MessageAction, or in the Message class. Changing the MessageAction seems better because the name is better. However, since MessageAction is an event, this probably means that when we restore an older session, the deserialization will fail (because the event was saved with `images_urls`).

We have a few options:
- ignore it; I don't know how much use or important older sessions restores are there yet, although it's default with UI; there's work ahead that appears to break them though
- change the other one instead
- add a small line somewhere like `events/deserialization/event.py` (from dict) to convert from `images_urls` to `image_urls`.

How about we do the latter? @Mr-Imperium

Reply:
Sure, working on it.